<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KQtNetworkTestClass</class>
 <widget class="QMainWindow" name="KQtNetworkTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KQtNetworkTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,2">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>http://119.45.4.14/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="kqtnetworktest.qrc"/>
 </resources>
 <connections/>
</ui>
